### ISSUE TITLE
Locks remain in the left bar after agreement

### ISSUE THREAD

**Issue by sntmoodle** [academic-moodle-cooperation/moodle-mod_consentform#11]
If our students agree to the consent form, the locks remain in the left bar of Moodle. And the operation is also blocked accordingly. See printscreen. Students must log out and log in again, to completely remove the blockage. 
Is there a solution?
![image](https://github.com/academic-moodle-cooperation/moodle-mod_consentform/assets/135450105/5bd6c5fb-5159-478c-9618-57fe1bd8db0f)

**Comment by bluetom** [comment]
No, a logout is absolutely not necessary. This problem is totally new to us. Maybe it has something to do with your (course-)cache management. If so, there is nothing the plugin can do to fix it. I will have a look into it...

**Comment by karl91aw** [comment]
Issue seems to be a Browser/Caching Problem.
